entrypoint.sh

```diff
@@ -34,34 +34,34 @@ HIGH)
   ;;
 esac
 
-if [ "$4" == "DEFAULT" ]; then
+if [ "$4" = "DEFAULT" ]; then
     EXCLUDED_PATHS=""
 else
     EXCLUDED_PATHS="-x $4"
 fi
 
-if [ "$5" == "DEFAULT" ]; then
+if [ "$5" = "DEFAULT" ]; then
     EXIT_ZERO=""
 else
     EXIT_ZERO="--exit-zero"
 fi
 
-if [ "$6" == "DEFAULT" ]; then
+if [ "$6" = "DEFAULT" ]; then
     SKIPS=""
 else
     SKIPS="-s $6"
 fi
 
-if [ "$7" == "DEFAULT" ]; then
+if [ "$7" = "DEFAULT" ]; then
     INI_PATH=""
 else
     INI_PATH="--ini $7"
 fi
 
 # select unique files/directories
-unique_directories=($(for dir in "${1[@]}"; do echo "${dir}"; done | sort -u))
+unique_directories=$(for dir in "$@"; do echo "$dir"; done | sort -u)
 
 # run bandit on each unique files/directories
-for dir in "${unique_directories[@]}"; do
-    bandit -r $dir $LEVEL $CONFIDENCE $EXCLUDED_PATHS $EXIT_ZERO $SKIPS $INI_PATH
+for dir in $unique_directories; do
```

Comment:
I think that the original is safer as it can handle spaces and glob characters such as `*`